src/libraries/System.Runtime.InteropServices.JavaScript/src/Resources/Strings.resx

```diff
@@ -129,6 +129,21 @@
   <data name="TypedArrayNotCorrectType" xml:space="preserve">
     <value>TypedArray is not of correct type.</value>
   </data>
+  <data name="MissingEntrypoint" xml:space="preserve">
```

Comment:
Maybe `MissingManagedEntrypointHandle`, and `Managed entrypoint handle is not set`?

Reply:
I created separate PR for those and others. https://github.com/dotnet/runtime/pull/79754